theme.html

```diff
@@ -0,0 +1,284 @@
+  }
+}, 1000);
+</script>
+
+  <section id="results">
+    <div class="d-md-flex flex-md-equal w-100 my-md-3 pl-md-3">
+      <div class="bg-light mr-md-3 overflow-hidden pl-4 pr-4">
+
+        <div class="mx-auto bd-highlight px-3">
+            
+        <div class="bs-example">
+            <ul class="nav nav-tabs">
+                <li class="nav-item">
+                    <a data-toggle="tab" id="home-tab" href="#home" class="nav-link active" role="tab" aria-controls="home" aria-selected="true">Theme of the Hackathon</a>
+                </li>
+                <li class="nav-item">
+                    <a data-toggle="tab" id="menu1-tab" href="#menu1" class="nav-link" role="tab" aria-controls="menu1" aria-selected="false">Submission Guidelines</a>
+                </li>
+                <li class="nav-item">
+                    <a data-toggle="tab" id="menu2-tab" href="#menu2" class="nav-link" role="tab" aria-controls="menu2" aria-selected="false">Judging Criteria</a>
+                </li>
+            </ul>
+        </div>
+<div class="tab-content">
+    <div id="home" class="tab-pane fade show active pt-2" role="tabpanel" aria-labelledby="home-tab">
+       <p class="text-justify">Synfig Studio 2D Animation Hackathon. The theme for animation is:</p>
+        <center><img src="image/patch2.png"><br><br></center>
+        <!-- <div class="container">
+        <center>  <img src="image/patch-bg.png" alt="Snow" style="width:50%;">
+          <div class="bottom">The Panchatantra Stories</div>
+        </div> -->
+
+
+    <p><span >The Panchtantra is an ancient Sanskrit collection of stories. Team should create a 2D animation for these stories, using Synfig Studio software.</span></p>
+    <p><span style="font-weight: 400;">Please find the list of Panchatantra stories along the moral </span><a href="https://docs.google.com/spreadsheets/d/1-xssRyZwSbva25zyLBlV2iWCDj4wx8PnCuEzFS-Uqiw/edit#gid=0" target="_blank"><span style="font-weight: 400;">here</span></a><span style="font-weight: 400;">.</span></p>
+    <p>You can have the stories interpreted according to the current situations happening across the world. For example the climate change, elections, pandemic situation, air pollution, etc. Please make sure that the moral of the story is not changed.</p>
+    </div>
+
+
+
+    <div id="menu1" class="tab-pane fade" role="tabpanel" aria-labelledby="menu1-tab">
+     <p>The participants have to submit their animation at <a href="https://courses.fossee.in">https://courses.fossee.in</a>. After confirming your participation in the Hackathon, your login credentials for submission will be generated and sent to you on the registered email ID.</p>
+        
+      <p><mark><span style="width:500px;height:100px;border:1px solid #000;">Please note: An email is sent to your registered email ID, where you have to confirm your participation in this Hackathon. Please make sure that this form is filled only by the team lead.</span></mark></p>
```

Comment:
Remove the border to this and any internal css applied

Reply:
Done